### ISSUE TITLE
Conversion error: `undefined variable module`

### ISSUE THREAD

**Issue by justinvp** [pulumi/pulumi-terraform-bridge#1147]
Error converting https://github.com/philips-labs/terraform-aws-github-runner:

```
pulumi convert --from terraform --language typescript --out pulumi
warning: Plugin converters are currently experimental
error: main.pp:104,1-146,2: main.pp:2,42-48: undefined variable module; ; main.pp:2,42-48: undefined variable module;
error: main.pp:237,1-270,2: runner-binaries-syncer.pp:0,37-43: undefined variable module; , and 4 other diagnostic(s); runner-binaries-syncer.pp:0,37-43: undefined variable module; , and 4 other diagnostic(s)
error: main.pp:147,1-236,2: main.pp:6,55-61: undefined variable module; , and 10 other diagnostic(s); main.pp:6,55-61: undefined variable module; , and 10 other diagnostic(s)
error: could not generate output program: failed to bind program
```

**Comment by Frassle** [comment]
Found it. This is trying to use the `path.module` built in variable from terraform. We're currently translating that as just `module`.

**Comment by Frassle** [comment]
Fixed by #1172
